### DISCUSSION TITLE
建议人体，人体存在，门窗什么的用二进制传感器的形式表示

### DISCUSSION THREAD

**Discussion by redhatme** [XiaoMi/ha_xiaomi_home#485] [Ideas]
建议人体，人体存在，门窗什么的用二进制传感器的形式表示，这样做自动化也简单。转homekit也可以直接转过去，不用通过模板转换一遍。对小白用户也友好。

**Comment by topsworld** [comment]
后续会增加兼容选项，之所以这样设计，是门窗传感器状态和门的状态是反的。

**Comment by dldui-captain** [comment]
请问如何通过模板转换？
